README.md

```diff
@@ -1 +1,5 @@
-# python-sdk
\ No newline at end of file
+# Sei SDK (python)
+
+![Sei Logo](https://raw.githubusercontent.com/sei-protocol/sei-chain/master/assets/SeiLogo.png)
+
+The Sei SDK is a framework for builidng [Sei](https://github.com/sei-protocol/sei-chain) applications in Python. It provides helpers that interacts with the chain modules through Python. 
```

Comment:
Could you add a section here around requirements and instructions for installing? Similar to what Terra has 
https://github.com/terra-money/terra.py

Reply:
Yup added